test/test_jit.py

```diff
@@ -12737,7 +12790,7 @@ def fn():
             x.add_(torch.ones(2, 3))
             return x_view
 
-        self.checkScript(fn, (), profiling=False)
+        self.checkScript(fn, (), profiling=ProfilingMode.EXECUTOR)
```

Comment:
just a nit: but this flag seems like it says "I want you to use the full profiler", but what it means is "i want you to use the full profiler, and I am going to rerun the function multiple times to test it". I think the keyword should make this clear. maybe `profile_and_replay=True`.